### ISSUE TITLE
Can't create multiple forms for same model type on latest master

### ISSUE THREAD

**Issue by danconnell** [piceaTech/ember-rapid-forms#88]
I tried out the latest version of master (commit afca35ecbcfeee91602cb09f0569dfff05c94948), since I wanted the prompt fix, but I found a problem.

I have the following code:
```
{{#each sortedOptions as |option index|}}
  {{#em-form action='formSubmitted' formLayout='horizontal' model=option submitButton=false}}
    {{em-input type='text' property='option_text' label=(t 'functions.concat' first='Option' second=option.order_index)}}
    {{em-checkbox property='preselected' label='Preselected' class='form-group inline-form-group'}}
  {{/em-form}}
{{/each}}
```

This results in error messages: 
`Uncaught Error: Assertion Failed: Attempted to register a view with an id already in use: option_text`
`Uncaught Error: Assertion Failed: Attempted to register a view with an id already in use: preselected`

Seems like I can't loop over an array of the same model as it's not using unique IDs somewhere. This code works on 1.0.0-beta4. 

I'm also getting deprecation warnings about modifying properties inside a didInsertElement hook coming from the latest code in master here too.

**Comment by danconnell** [comment]
Seems to work fine if I set the cid's to something unique on the em-* inputs in the loop.

This seems like a bug, though.

**Comment by danconnell** [comment]
Is it possible that this is breaking input bindings? Even after adding cid's to the inputs in the loop so that this error doesn't happen, I get this weird behaviour where the inputs can't update the model.

If the property has text in it, the input is prepopulated, but any changes aren't reflected on the model. It's as if the model or the property is being treated as readonly. I know they shouldn't be, though, because if I do it outside of a loop on a computed property of the firstObject in the array, the bindings work fine.

Any ideas?

**Comment by GCorbel** [comment]
@danconnell , you're right. I made this bug and I will find the solution to solve it.

**Comment by GCorbel** [comment]
It will be fixed by #89 once merged.
